src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -438,10 +404,7 @@ public void ValidateXml(XmlSchemaSet schemas, ValidationEventHandler validationE
 
             using (var partStream = GetStream())
             {
-                if (validationEventHandler is not null)
-                {
-                    xmlReaderSettings.ValidationEventHandler += validationEventHandler;
-                }
+                xmlReaderSettings.ValidationEventHandler += validationEventHandler;
```

Comment:
why this change?

Reply:
`validationEventHandler` is not nullable, so the condition is always true.